VideoApi/VideoApi.IntegrationTests/packages.lock.json

```diff
@@ -3204,7 +3204,8 @@
         "type": "Project",
         "dependencies": {
           "Enums.NET": "4.0.0",
-          "System.ComponentModel.Annotations": "4.7.0"
+          "System.ComponentModel.Annotations": "4.7.0",
+          "VideoApi.Contract": "1.0.0"
```

Comment:
same as other projects we have, we are already referencing common and we can remove this reference once the const is moved